src/main/resources/application.properties

```diff
@@ -1,3 +1,6 @@
 spring.datasource.url=jdbc:mysql://localhost:3306/secur_auth
 spring.datasource.username=root
 spring.datasource.password=password
+
```

Comment:
Please change to use profiles for application.properties

Reply:
https://trello.com/c/7FJ8hjj6/62-sec-29-create-profiles-for-secur-auth